### PR TITLE
add demo video

### PR DESCRIPTION
**Pull request by uzurpastor**
Hi, I've added the demo version. Also in the process I found a bunch of errors on your demo page:
 - in focus/jumping mode, it is impossible to focus on the ‘list2’(`k`) button. 
 - the inherited ‘About’ menu is not displayed (fixed in this PR)
 - as stated [here](https://github.com/ai/keyux?tab=readme-ov-file#focus-jumps), the `esc` key should bring the focus area back. My understanding is that 
   - expected:
      - clear information about the last focused element
      - when pressing the `tab` key: jump to the first element on the page, 
   - current:
      - hiding the blue frame
      - when you press the `tab` key: jump to the next element

I think that's it. I hope my PR will be useful.

### CODE REVIEW

**Comment by ai** [comment]
1. Don't do different things in a single PR. Please open new PR with fixes. 
2. Open new issue with issue report. 
3. The panel with pressed key is not very easy to understand because it renders arrows as letters.

**Comment by uzurpastor** [comment]
1. yep
2. yep
3. Not sure if it be possible to implement from my side. I researched actual key showers for Linux systems and this one "[showmethekey](https://showmethekey.alynx.one/)" works good. But there is a bit unclear manual about how to run with utf8 encoding.  this issue would be actual tomorrow, I'll try then because it can take a time.

**Comment by ai** [comment]
> Not sure if it be possible to implement from my side. I researched actual key showers for Linux systems and this one "[showmethekey](https://showmethekey.alynx.one/)" works good. But there is a bit unclear manual about how to run with utf8 encoding. if the issue would be actual tomorrow, I'll try then because it can take a time.

Maybe this task is not for Linux users. I have Linux too and created this task because I don't know how to create a beautiful video in Linux

File: test/demo/index.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const Counter: FC = () => {
 const Search: FC = () => {
   return (
     <div>
-      <input aria-keyshortcuts="f" placeholder="f" type="text" />
+      <input aria-keyshortcuts="shift+s" placeholder="S" type="text" />
```
Comment (ai) [inline test/demo/index.tsx:89]: Yo have different hotkey in `aria-keyshortcuts` and `placeholder`

File: test/demo/index.tsx [hunk 2]
```diff
@@ -147,7 +147,7 @@ const Page: FC<{
     content = (
       <>
         <input aria-controls="results" placeholder="Search" type="search" />
-        <ul id="results" role="listbox">
+        <ul id="results" aria-keyshortcuts="i" role="listbox">
```
Comment (ai) [inline test/demo/index.tsx:150]: You should move focus here by pressing Enter in search field

**Comment by ai** [comment]
I am closing this PR in favor of https://github.com/ai/keyux/pull/28

Please open other PR with the fix and issue.

**Pull request closed without merging**